Sources/SwiftCrossUI/Backend/AppBackend.swift

```diff
@@ -1139,4 +1145,8 @@ extension AppBackend {
     ) {
         todo()
     }
+
+    public func limitScreenBounds(_ bounds: SIMD2<Int>) -> SIMD2<Int> {
```

Comment:
I feel like this should be reworked to `func clampWindowSize(_ size: SIMD2<Int>, for window: Window) -> SIMD2<Int>`. Renamed for clarity, and extra parameter added for future proofing (future purposes could want the window in question to access properties such as scaling factors).